### ISSUE TITLE
No arrow in environment pane for Data.Table object

### ISSUE THREAD

**Issue by Oravishayrizi** [rstudio/rstudio#4882]
I recently updated to Rstudio  1.2.1335 

Since than I don't see the right arrow in the environment pane for Data.Table object.
data.frame has this arrow.

Here is a code to reproduce my problem:
```
library(data.table)
data(iris)
class(iris)
dt<-as.data.table(iris)
class(dt) 
```
Checked in the options but nothing seems related, Am I missing something?
![image](https://user-images.githubusercontent.com/30078665/58752369-6383b100-84b6-11e9-87c4-8c9d1641998e.png)

**Comment by pascaloettli** [comment]
I think it is related to #4171 #4758 and #4791. This is fixed in RStudio v1.3

![Screenshot from 2019-06-03 10-47-42](https://user-images.githubusercontent.com/6613954/58771039-12c2a400-85ed-11e9-835a-b228b9a1a4b3.png)

**Comment by ronblum** [comment]
@Oravishayrizi Thank you for raising the issue. As @pascaloettli shows, this appears to be related to the other issues. I double-checked the example you provided, and it works in 1.3.

If you're interested, we have[ daily builds available for RStudio 1.3](https://dailies.rstudio.com/). However, these are intended for testing purposes at this stage.
